picard/ui/metadatabox.py

```diff
@@ -96,18 +96,19 @@ def display_value(self, tag):
 
 class TagDiff(object):
 
-    __slots__ = ("tag_names", "new", "orig", "status", "objects")
+    __slots__ = ("tag_names", "new", "orig", "status", "objects", "max_length_delta_ms")
 
-    def __init__(self):
+    def __init__(self, max_length_diff=4):
```

Comment:
Do we really need this to be a KW argument? Any reason why we are not just setting the value below at line 107 from config.setting ? It has a default value of 2, so it won't be undefined anytime?

Reply:
Also, if it _is_ to be a KW argument, the default should probably be 2.

Reply:
> Do we really need this to be a KW argument? Any reason why we are not just setting the value below at line 107 from config.setting ? It has a default value of 2, so it won't be undefined anytime?

I prefer to make this argument optional with a safe default.